app/models/ability.rb

```diff
@@ -25,6 +25,14 @@ def initialize(user)
       user.admin? or signed_in?(user) && team.new_record? or on_team?(user, team)
     end
 
+    can :update_attendances, Team do |team|
+      user.admin? or (user.student? && (team.students.include? user))
+    end
+
+    can :confirm_attendance, Team do |team|
```

Comment:
no, but this one is to team confirm its attendance, not its preference: 
![attend](https://user-images.githubusercontent.com/15239141/28418633-c1311178-6d32-11e7-8ee5-30aa135e901e.png)
We check the ability to confirm attendance in the teams show view. When some user access the show team page, we check if the user can confirm attendance. If it can, we show these links.